### ISSUE TITLE
"Query asks for data past end of contig" during report creation

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/vip#351]
VIP pipeline can fail with the following error:
```
Command output:
  creating report for 'vip.vcf.gz' ...
  Query asks for data past end of contig
```

Update `echo -e "params { vcf.filter.classes = \"B,LB,VUS,LP,P\"\nvcf.filter_samples.classes = \"LQ,MV,OK\" }" > "${OUTPUT_DIR}/custom.cfg"` to `echo -e "params { vcf.filter.classes = \"LQ,B,LB,VUS,LP,P\"\nvcf.filter_samples.classes = \"LQ,MV,OK\" }" > "${OUTPUT_DIR}/custom.cfg"` in `test_fastq_pacbio_hifi` in `test_fastq.sh`

**Comment by dennishendriksen** [comment]
caused by https://github.com/molgenis/vip-report/issues/122

**Comment by dennishendriksen** [comment]
Fixed in v5.5.0
